infrastructures/openshift/src/test/java/org/eclipse/che/workspace/infrastructure/openshift/environment/OpenShiftEnvironmentParserTest.java

```diff
@@ -48,48 +50,53 @@
 import org.testng.annotations.Test;
 
 /**
- * Tests {@link OpenShiftEnvironmentParser}.
+ * Tests {@link OpenShiftEnvironmentFactory}.
  *
  * @author Anton Korneta
  */
 public class OpenShiftEnvironmentParserTest {
```

Comment:
Let's rename test class according to the renaming of class that is tested here. 
`OpenShiftEnvironmentParserTest` -> `OpenShiftEnvironmentFactoryTest`